evennia/server/serversession.py

```diff
@@ -382,57 +397,6 @@ def at_cmdset_get(self, **kwargs):
 
         pass
 
-    # Mock db/ndb properties for allowing easy storage on the session
-    # (note that no databse is involved at all here. session.db.attr =
-    # value just saves a normal property in memory, just like ndb).
-
-    @lazy_property
-    def nattributes(self):
-        return AttributeHandler(self, InMemoryAttributeBackend)
-
-    @lazy_property
-    def attributes(self):
-        return self.nattributes
-
-    # @property
-    def ndb_get(self):
-        """
-        A non-persistent store (ndb: NonDataBase). Everything stored
-        to this is guaranteed to be cleared when a server is shutdown.
-        Syntax is same as for the _get_db_holder() method and
-        property, e.g. obj.ndb.attr = value etc.
-
-        """
-        try:
-            return self._ndb_holder
-        except AttributeError:
-            self._ndb_holder = DbHolder(self, "nattrhandler", manager_name="nattributes")
-            return self._ndb_holder
-
-    # @ndb.setter
-    def ndb_set(self, value):
-        """
-        Stop accidentally replacing the db object
-
-        Args:
-            value (any): A value to store in the ndb.
-
-        """
-        string = "Cannot assign directly to ndb object! "
-        string += "Use ndb.attr=value instead."
-        raise Exception(string)
-
-    # @ndb.deleter
-    def ndb_del(self):
-        """
-        Stop accidental deletion.
-
-        """
-        raise Exception("Cannot delete the ndb object!")
-
-    ndb = property(ndb_get, ndb_set, ndb_del)
-    db = property(ndb_get, ndb_set, ndb_del)
```

Comment:
Why are these removed?

Reply:
I'm not sure why that happened. I think I was meaning to create some kind of mixin to deduplicate some code and I got distracted.